CHANGELOG.rst

```diff
@@ -165,25 +195,9 @@ New Modules
 v1.0.3
 ======
 
-Release Summary
```

Comment:
the tool is removing this for iosxr also I manually added that again

Reply:
manually added back @ashwini-mhatre thanks